src/lib.rs

```diff
@@ -162,6 +163,13 @@ impl<T: Publisher + Clone + 'static> DeviceManager<T> {
                             .telemetry_config_event(interface_name, endpoint, data)
                             .await;
                     }
+                    (
+                        "io.edgehog.devicemanager.LedBehavior",
+                        [led_id, "behavior"],
+                        Aggregation::Individual(AstarteType::String(behavior)),
+                    ) => {
+                        led_behavior::set_behavior(led_id, behavior).await;
```

Comment:
I'm not sure that we have to await the end of `set_behavior` function, because could be a lot of repetitions, maybe is better to move in another asynchronous task.